docs/reference/landscape.md

```diff
@@ -57,6 +57,8 @@ account_name = standalone
 log_level = debug
 ```
 
+For more information, see [the Landscape guide on how to configure Ubuntu Pro for WSL](https://ubuntu.com/landscape/docs/register-wsl-hosts-to-landscape#heading--configure-ubuntu-pro-for-wsl-for-landscape).
```

Comment:
```suggestion
For more information, see [the Landscape guide on how to configure Ubuntu Pro for WSL](https://ubuntu.com/landscape/docs/register-wsl-hosts-to-landscape/#heading--configure-ubuntu-pro-for-wsl-for-landscape).
```

Linkcheck did fail for me as well. Seems like adding a `/` before the anchor solves the issue.